engine/src/deploy.cpp

```diff
@@ -531,6 +582,51 @@ Exec_stat MCIdeDeploy::exec(MCExecPoint& ep)
 		if (t_stat == ES_NORMAL && ep2 . getarray() != NULL)
 			t_params . version_info = new MCVariableValue(*ep2 . getarray());
 	}
+    
+    // The 'min_os_version' is either a string or an array. If it is a string then
+    // it encodes the version against the 'Unknown' architecture which is interpreted
+    // by the deploy command to mean all architectures. Otherwise, the keys in the
+    // array are assumed to be architecture names and each is pushed on the array.
+    // If multiple entries are present, then the 'unknown' mapping is used for any
+    // architecture not explicitly specified. The current architecture strings that are
+    // known are:
+    //   i386, x86_64, armv6, armv7, armv7s, arm64, ppc, ppc64
+    // The empty string is taken to be 'unknown'.
+    if (t_stat == ES_NORMAL)
+    {
+        t_stat = t_array -> fetch_element(ep2, "min_os_version");
+        if (t_stat == ES_NORMAL)
```

Comment:
True - but it was more to make it clear what code is manipulating the 'min_os_version' field - its all contained in the outer { ... }.